src/main/java/seedu/address/model/tutee/Payment.java

```diff
@@ -17,15 +17,19 @@
 public class Payment {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Payment values should only contain numbers, and it should be at least 1 digit long.";
+            "Payment values should only contain positive numbers with at least 1 digit, allowing 0 or 2 decimals i.e"
+                    + " 100 or 74.50";
+    public static final String DECIMAL_CONSTRAINTS =
+            "Payment values must have either 0 or 2 decimal places and end with either a 0 or 5, i.e 40.50 or 40.55.";
```

Comment:
For this portion, is it possible to change it to 
```suggestion
            "Payment values must have either 0 or 2 decimal places. If it is a decimal, it has to end with either a 0 or 5, i.e 40.50 or 40.55.";
```

Reply:
Good suggestion. I've simplified it further by making it:
`"Payment values must have either 0 or 2 decimal places. If it has 2 decimal places, it must end with either a 0 or 5, i.e 40.50 or 40.55.";`